README.md

```diff
@@ -1,2 +1,8 @@
 # Hello-World
 Just my first repository
+Hi, my name is betsaida, I am 23 years old, I am dominican, I want to be a master software developer,
+but now a very confused with a lot of tecnology, I good at logical, but I don't know How translate this intro
+real world, now for example I want to create a project , I know what I want, I know how it will look like, but
+I don't have the idea how to do it, What i have to uses, since cero.
+
+I am here looking for an orientation...
```

Comment:
I been working with access for one years, I developed so project here, but the emigration form here is necessary and so frustration for me